### ISSUE TITLE
0.18.2 build error: Docks/GeoImageDock.h:7:33: fatal error: QtWidgets/QDockWidget: No such file or directory

### ISSUE THREAD

**Issue by turion** [openstreetmap/merkaartor#65]
When building 0.18.2 on Gentoo, there is an error during the build:

```
In file included from Docks/PropertiesDock.cpp:23:0:
Docks/GeoImageDock.h:7:33: fatal error: QtWidgets/QDockWidget: No such file or directory
 #include <QtWidgets/QDockWidget>
```

**Comment by sebastic** [comment]
This issue is closely related to #60 (maybe even a duplicate).

You can include https://github.com/openstreetmap/merkaartor/commit/e6d015a953a163f174e09a9b50c9bb8bc19593e4 as a patch in the Gentoo ebuild, Fedora and Debian have included this patch for Merkaartor 0.18.2 in their packaging already (I'm not sure about other distributions).

**Comment by Krakonos** [comment]
As the e6d015a fixes the issue, I consider it fixed. Please, comment further if it's not true.
